### PR TITLE
Vega docs: add note about file type

### PR DESCRIPTION
**Pull request by fiskus**
- [x] Documentation

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203858164330901

### CODE REVIEW

File: docs/Catalog/VisualizationDashboards.md [hunk 1]
```diff
@@ -115,7 +115,19 @@ file with a library-compatible schema in a JSON file as follows in your
 ```
 
 For both Vega and Vega Lite you may specify relative paths to package files
-as data sources and the Quilt catalog will resolve the same.
+as data sources and the Quilt catalog will resolve the same. Vega treats this
```
Comment (robnewman) [inline docs/Catalog/VisualizationDashboards.md:118]:
```suggestion
as data sources and the Quilt catalog correctly resolves them. Vega treats any
```

File: docs/Catalog/VisualizationDashboards.md [hunk 2]
```diff
@@ -115,7 +115,19 @@ file with a library-compatible schema in a JSON file as follows in your
...
+data source as JSON by default. Please,
+[specify the file type](https://vega.github.io/vega-lite/docs/data.html#format)
+if you want to use anything than JSON. Example:
```
Comment (robnewman) [inline docs/Catalog/VisualizationDashboards.md:121]:
```suggestion
data source as JSON by default. If you wish to use a different format than JSON, please 
[specify the file type](https://vega.github.io/vega-lite/docs/data.html#format).
For example: 
```

**Review by robnewman — DISMISSED** [review]
Approved subject to merging in suggested changes.

File: docs/Catalog/VisualizationDashboards.md [hunk 3]
```diff
@@ -115,7 +115,19 @@ file with a library-compatible schema in a JSON file as follows in your
...
+[specify the file type](https://vega.github.io/vega-lite/docs/data.html#format)
+if you want to use anything than JSON. Example: 
+
+```json
+{
+  "data": "./datasource.csv"
+  "format": {
+    "type": "csv"
+  }
+}
```
Comment (robnewman) [inline docs/Catalog/VisualizationDashboards.md:129]:
```suggestion
{
  "data": {
    "url": "./datasource.csv",
    "format": {
      "type": "csv"
    }
}
```

Comment (fiskus) [reply docs/Catalog/VisualizationDashboards.md:129]: oops

**Review by robnewman — APPROVED** [review]

**Pull request merged**